FourthTask/figures/types/types.go

```diff
@@ -0,0 +1,70 @@
+//	import $GOPATH/src/github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/types
+//	Package contains structures descriptions and constructors, interface definition
+
+package types
+
+import (
+	"errors"
+	"math"
+)
+
+const (
+	circlePanic   string = " circle can't has negative radius"
```

Comment:
Common practice:
```suggestion
	ErrCircleNegative = errors.New( " circle can't has negative radius")
```
And it will be var, not const
Also, there are some other ways to define errors like this one, feel free to use any you will find suitable.